site/docs/hive.md

```diff
@@ -72,6 +72,15 @@ catalog.createTable(tableId, schema, spec, tableProperties);
 
 The table level configuration overrides the global Hadoop configuration.
 
+#### Hive on Tez configuration
+
+To use the Tez engine on Hive(requires version >= `3.1.2`), Tez needs to be upgraded to >= `0.10.1` which contains a necessary fix [Tez-4248](https://issues.apache.org/jira/browse/TEZ-4248).
```

Comment:
The wording is a tiny bit confusing to me. `(requires version >= 3.1.2)` would indicate to me that Tez can only be used with Hive 3. Shall we reword it like this?
`To use the Tez engine on Hive 3.1.2 or later, ...`
followed by: 
`To use the Tez engine on Hive 2.3.x, ...`

Reply:
Yep. That would be much more clear. 
Updated the PR. Please take another look.

Reply:
Looks good now, thanks